aspnet/fundamentals/localization.rst

```diff
@@ -1,8 +1,302 @@
-.. include:: /../common/stub-topic.txt
+ASP.NET Core Internationalization, Localization and Globalization 
+==================================================================
 
-|stub-icon| Localization
-========================
+By `Rick Anderson`_ with assistance from `Damien Bowden <https://twitter.com/damien_bod>`_, `Bart Calixto @bartmax <https://twitter.com/bartmax>`_, and `Nadeem Afana <https://twitter.com/NadeemAfana>`__ 
 
-.. include:: /../common/stub-notice.txt
+Creating a multilingual website withe ASP.NET Core is not easy but it will allow your site to reach a wider audience. ASP.NET  has :doc:`/fundamentals/request-features` that provide support for localizing into different languages and cultures.
+
+This sample is based on the ``Localization.StarterWeb`` project on `GitHub <https://github.com/aspnet/entropy>`__.
+
+.. contents:: In this article:
+  :local:
+  :depth: 1
+  
+.. note:: Currently, resource files are not read when the project is run from Visual Studio. See `this issue <https://github.com/aspnet/dnx/issues/3047>`_ for more information.
```

Comment:
It's probably obvious to most after knowing that it won't work in VS, but you might want to include a little something telling them that in the mean time to test localization you run it from the command line.